vars/main.yml

```diff
@@ -331,6 +332,7 @@ pgbackrest_conf:
     - {option: "log-level-file", value: "detail"}
     - {option: "log-path", value: "/var/log/pgbackrest"}
     - {option: "repo1-type", value: "{{ pgbackrest_repo_type |lower }}"}
+#    - {option: "repo1-retention-full", value: 5} #  Take care, not to fill up your disks with backups
```

Comment:
I think it's worth uncommenting out this parameter, but specify the default value of 1 or 2

> Take care, not to fill up your disks with backups

this comment seems superfluous to me